src/tests/topp/QCCalculator_2_output.mzQC

```diff
@@ -0,0 +1,92 @@
+{
+  "MzQC": {
```

Comment:
can you check or validate somehow against the schema?
e.g., MzQC should be mzQC see https://github.com/HUPO-PSI/mzQC/blob/master/schema/mzqc_schema.json